uint16 time
---
string flag

